app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurancedetail/yourinfo/YourInfoTab.kt

```diff
@@ -156,8 +156,6 @@ internal fun YourInfoTab(
         )
       }
     }
-
-    Spacer(Modifier.height(8.dp))
```

Comment:
Was this part of the UI review too? From looking at the code surrounding this it feels like the coverage rows  below would then be too close to the optional notification card  above no?